crates/goose/src/providers/anthropic.rs

```diff
@@ -48,76 +54,82 @@ impl Default for AnthropicProvider {
 impl AnthropicProvider {
     pub fn from_env(model: ModelConfig) -> Result<Self> {
         let config = crate::config::Config::global();
-        let api_key: String = config.get_secret("ANTHROPIC_API_KEY")?;
-        let host: String = config
-            .get_param("ANTHROPIC_HOST")
```

Comment:
should this still be used - allows people to override the host (not sure if anyone does though)